cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/LocalizationUtils.java

```diff
@@ -42,6 +48,43 @@
 public final class LocalizationUtils {
   private static final Logger LOG = LoggerFactory.getLogger(LocalizationUtils.class);
 
+  public static List<File> getExtraJars(CConfiguration cConf) {
+    String extraJarConf = cConf.get(Constants.AppFabric.EXTRA_JARS);
+    List<String> extraJars = extraJarConf == null ?
+      new ArrayList<String>() : Arrays.asList(extraJarConf.split(","));
+    List<File> jarFiles = new ArrayList<>();
+    for (String jarPath : extraJars) {
+      jarFiles.add(new File(jarPath));
+    }
+    return jarFiles;
+  }
+
+  public static  List<String> getExtraJarNames(CConfiguration cConf) {
```

Comment:
Do you think it will be better to move these two methods to `CConfiguration` as a member method?